Editor/PropertyDatabase/PropertyDatabaseWindow.cs

```diff
@@ -0,0 +1,342 @@
+#if USE_PROPERTY_DATABASE
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor.UIElements;
+using UnityEngine;
+using UnityEngine.UIElements;
+using UnityEditor.Search;
+using UnityEditor;
+
+class PropertyDatabaseWindow : EditorWindow
+{
+    private const string k_PropertyDatabaseWindowAssetPath = "Packages/com.unity.search.extensions/Editor/UXML/PropertyDatabase/PropertyDatabaseWindow.uxml";
+    private const string k_PropertyDatabaseListViewAssetPath = "Packages/com.unity.search.extensions/Editor/UXML/PropertyDatabase/PropertyDatabaseListView.uxml";
+    private const string k_PropertyDatabaseItemAssetPath = "Packages/com.unity.search.extensions/Editor/UXML/PropertyDatabase/PropertyDatabaseItem.uxml";
+    private const string k_PropertyStringListViewAssetPath = "Packages/com.unity.search.extensions/Editor/UXML/PropertyDatabase/PropertyStringListView.uxml";
+    private const string k_PropertyStringItemAssetPath = "Packages/com.unity.search.extensions/Editor/UXML/PropertyDatabase/PropertyStringItem.uxml";
+
+    private static string s_PropertyDatabaseFileName;
+
+    private static PropertyDatabaseView s_PropertyDatabaseView;
+    private static TemplateContainer s_PropertyDatabaseWindowUI;
+
+    private static bool s_CreatedFromPropertyDatabaseView = false;
```

Comment:
This is fine for now, but I would prefer if we would not use static member for the state of the window, in case we want to support opening multiple windows in the future.